packages/pds-ember/app/styles/pds/components/select/_control/_config.scss

```diff
@@ -24,6 +24,7 @@ $padding--inline: theme.$size--sm; // ~12px
   color: custom.$color;
   font-family: $fontFamily;
   font-size: $fontSize;
+  line-height: inherit;
```

Comment:
The `<select>` will inherit `line-height: 1` from the component wrapper.  It's okay to do this, because browsers don't render multi-line options (see comment on `tests/dummy/app/components/docs/form-field/select.hbs`).